sdk/bootstrap/main.go

```diff
@@ -251,6 +260,49 @@ func renderTemplates(templateMaxBytes int64) {
 	}
 }
 
+func installCerts() {
+	mesosSandbox := os.Getenv("MESOS_SANDBOX")
+	sslDir := filepath.Join(mesosSandbox, ".ssl");
+
+	sslDirExists, err := _pathExists(sslDir)
+	if !sslDirExists || err != nil {
```

Comment:
Can we log the err if there is one?  Or is this level of nitpicking foolish at this point in the project?